go-controller/pkg/ovn/policy.go

```diff
@@ -206,6 +239,7 @@ func namespacePortGroupACLName(namespace, portGroup, name string) string {
 }
 
 func buildACL(namespace, portGroup, name, direction string, priority int, match, action, aclLogging string, policyType knet.PolicyType) *nbdb.ACL {
```

Comment:
nit:
We could have a `buildACLForPolicyType` that would derive direction, (some) external ids and (some) options from the policy type, and then call this buildACL with that info. `buildACLForPolicyType` could also be used from `gressPolicy` buildACL. This buildACL would no longer have a policy type argument. 

So an attempt to unify basic ACL building logic based on policy type:

`gressPolicy.buildACL() -> buildACLForPolicyType() -> buildACL() -> libovsdbops.buildACL()`

Reply:
created an issue to unify ACL building logic: https://github.com/ovn-org/ovn-kubernetes/issues/2892